src/component/feature/FeatureToggleList/FeatureToggleListTable/FeatureToggleListTable.tsx

```diff
@@ -0,0 +1,196 @@
+import { TableToolbar } from 'component/common/Table/TableToolbar/TableToolbar';
+import TablePlaceholder from 'component/common/Table/TablePlaceholder/TablePlaceholder';
+import { SearchHighlightProvider } from 'component/common/Table/SearchHighlightContext/SearchHighlightContext';
+import { DateCell } from './DateCell/DateCell';
+import { FeatureNameCell } from './FeatureNameCell/FeatureNameCell';
+import { FeatureSeenCell } from './FeatureSeenCell/FeatureSeenCell';
+import { FeatureStaleCell } from './FeatureStaleCell/FeatureStaleCell';
+import { FeatureTypeCell } from './FeatureTypeCell/FeatureTypeCell';
+import { LinkCell } from './LinkCell/LinkCell';
+import { CreateFeatureButton } from '../../CreateFeatureButton/CreateFeatureButton';
+import { ConditionallyRender } from 'component/common/ConditionallyRender/ConditionallyRender';
+
+interface IExperimentProps {
+    data: Record<string, any>[];
+    isLoading?: boolean;
+}
+
+const sortTypes = {
+    date: (a: any, b: any, id: string) =>
+        b?.values?.[id]?.getTime() - a?.values?.[id]?.getTime(),
+    boolean: (v1: any, v2: any, id: string) => {
+        const a = v1?.values?.[id];
+        const b = v2?.values?.[id];
+        return a === b ? 0 : a ? 1 : -1;
+    },
+    alphanumeric: (a: any, b: any, id: string) =>
+        a?.values?.[id]
+            ?.toLowerCase()
+            .localeCompare(b?.values?.[id]?.toLowerCase()),
+};
+
+const columns = [
+    {
+        Header: 'Seen',
+        accessor: 'lastSeenAt',
+        Cell: FeatureSeenCell,
+        sortType: 'date',
+    },
+    {
+        Header: 'Type',
+        accessor: 'type',
+        Cell: FeatureTypeCell,
+    },
+    {
+        Header: 'Feature toggle name',
+        accessor: 'name',
+        // @ts-expect-error // TODO: props type
+        Cell: ({ row: { original } }) => <FeatureNameCell {...original} />,
+        sortType: 'alphanumeric',
+    },
+    {
+        Header: 'Created on',
+        accessor: 'createdAt',
+        Cell: DateCell,
+        sortType: 'date',
+    },
+    {
+        Header: 'Project ID',
+        accessor: 'project',
+        Cell: ({ value }: { value: string }) => (
+            <LinkCell to={`/projects/${value}`}>{value}</LinkCell>
+        ),
+        sortType: 'alphanumeric',
+    },
+    {
+        Header: 'State',
+        accessor: 'stale',
+        Cell: FeatureStaleCell,
+        sortType: 'boolean',
+    },
+    // Always hidden -- for search
+    {
+        accessor: 'description',
+    },
+];
+
+export const FeatureToggleListTable: VFC<IExperimentProps> = ({
+    data: input,
+    isLoading = false,
+}) => {
+    const data = useMemo(() => input, [input]);
+    const theme = useTheme();
+    const isSmallScreen = useMediaQuery(theme.breakpoints.down('md'));
+    const isMediumScreen = useMediaQuery(theme.breakpoints.down('lg'));
+    const ref = useLoading(isLoading);
+
+    const initialState = useMemo(
```

Comment:
Does this need to be memoized?

Reply:
From what I read in `react-table` docs, it's important to memoize some props for performance reasons, and this is one of items that shouldn't change